src/status_im/utils/universal_links/core.cljs

```diff
@@ -76,11 +75,21 @@
    (chat/start-public-chat public-chat {})
    (pairing/sync-public-chat public-chat)))
 
-(fx/defn handle-view-profile [{:keys [db] :as cofx} public-key]
-  (log/info "universal-links: handling view profile" public-key)
-  (if (new-chat.db/own-public-key? db public-key)
+(fx/defn handle-view-profile
```

Comment:
If the public-key is wrong it navigates to a bad screen. Noticed that but havent't fixed it in current PR, not sure if there is a task for that @hesterbruikman

Reply:
@Ferossgp what do you mean with 'public key' is wrong? And what screen do you get? Don't recall an issue for this.

Reply:
I mean if a user does not exist with that pub-key. You get this screen with an empty name, then when you try to send a message it shows for a second an error.
![Screenshot 2020-01-16 at 12 50 53](https://user-images.githubusercontent.com/5794620/72514057-0844da00-385f-11ea-8107-1512626be740.png)

Reply:
Let me try to reproduce. Can you share the steps that resulted in a non-existent public key? Puzzles me because public key is the user identifier. I can't imagine a way to get it if it doesn't exist.

Reply:
The experiment was with a broken link, for example, you can remove the last symbol from the link. This mimics a user error when the link is shared.